pkg/services/validationService.go

```diff
@@ -3,13 +3,15 @@ package services
 import (
 	"encoding/json"
 	"fmt"
-	"github.com/datreeio/admission-webhook-datree/pkg/k8sMetadataUtil"
-	"github.com/datreeio/datree/pkg/deploymentConfig"
 	"net/http"
 	"os"
+	"strconv"
```

Comment:
We also need to send to the backend in POSt `/policyCheck/result` the field `cli_evaluation.metadata.clusterContext.isEnforceMode`